docs/source/asr/speaker_diarization/configs.rst

```diff
@@ -3,13 +3,10 @@ NeMo Speaker Diarization Configuration Files
 
 Since speaker diarization model here is not a fully-trainble End-to-End model but an inference pipeline, we use **diarizer** instead of **model** which is used in other tasks.
 
-The diarizer section will generally require information about the dataset(s) being
-used, models used in this pipline, as well as inference related parameters such as postprocessing of each models.
+The diarizer section will generally require information about the dataset(s) being used, models used in this pipline, as well as inference related parameters such as postprocessing of each models.
```

Comment:
pipline -> pipeline
postprocessing -> post processing

Reply:
Fixed. BTW, what would be the example of the post processing?

Reply:
It was for VAD actually.